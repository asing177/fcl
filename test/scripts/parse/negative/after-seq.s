// Ensure that sequencing in `after` guard expressions doesn't parse
// since this assumption is made in the type checker, e.g. we don't take
// scoping into account because we can't assign any new temporary variables

@initial
foo() {
  after (tmp = "2015-10-10T00:00:00Z"; tmp) {
    transitionTo(@terminal)
  }
}
