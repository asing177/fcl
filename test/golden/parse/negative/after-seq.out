foo() {
  after (tmp = "2015-10-10T00:00:00Z"; tmp) {
-------------------------------------^
    transitionTo(@terminal)
  }

"<stdin>" (line 7, column 38):
unexpected ";"
expecting operator or ")"